apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/PatientLinkBuilderTest.java

```diff
@@ -132,7 +132,7 @@ public void testMdcLogsInAddResourcesToBundle() {
     TransformerUtils.addResourcesToBundle(
         bundle,
         Collections.singletonList(
-            new ExplanationOfBenefit().setPatient(new Reference("Patient/Id"))));
+            new ExplanationOfBenefit().setPatient(new Reference("Patient/567834"))));
     assertTrue(bundle.getLink().isEmpty());
     paging.addLinks(bundle);
     assertNotNull(bundle.getLink(Constants.LINK_SELF));
```

Comment:
general comment -- we should write some tests to verify that bene_id gets set in the MDC for each call

Reply:
Just added unit tests